examples/todoapp/bin/run_integration_tests.dart

```diff
@@ -0,0 +1,156 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:collection/collection.dart';
+
+Future<int> main(List<String> arguments) async {
+  final runner = CommandRunner<int>(
+      'run_integration_tests', 'Integration test helper tool')
+    ..addCommand(ExecuteCommand());
+
+  final exitCode = await runner.run(arguments).catchError((error) {
+    if (error is UsageException) {
+      runner.printUsage();
+      exit(64); // Exit code 64 indicates a usage error.
+    }
+    throw error;
+  });
+
+  return exitCode ?? -1;
+}
+
+class ExecuteCommand extends Command<int> {
+  @override
+  String get description => 'Run the integration tests';
+
+  @override
+  String get name => 'execute';
+
+  @override
+  FutureOr<int>? run() async {
+    final testSets = {
+      'all': 'all',
+      'regression tests': 'regression',
+      'negative tests': 'negative',
+      'positive tests': 'positive'
```

Comment:
I see the `regression`, `positive` and `negative` used as tags in other files. It would be better if we introduced them as enums or constants so they don't get mistyped.

Reply:
Changed to constants